src/api/app/views/webui2/webui/user/_involvement.html.haml

```diff
@@ -21,7 +21,7 @@
           %p.md-3
             #{user.name} is not involved in any packages
         - else
-          %table.table.table-striped.table-bordered#involved-packages-table
+          %table.responsible.table.table-striped.table-bordered#involved-packages-table
```

Comment:
`responsive` ??

Reply:
yes :see_no_evil:

Reply:
but if I write the correct one it also breaks. I think we have to remove the responsive thing. It has more bugs than what is doing correctly

Reply:
https://getbootstrap.com/docs/4.0/content/tables/#responsive-tables

Reply:
@hellcp It needs to be on the [container of the table](https://getbootstrap.com/docs/4.1/content/tables/#responsive-tables). But we don't use it in this case as the table doesn't overflow (so we don't need the scrollbar)

Reply:
@dmarcoux true, I didn't even look what kind of table that was :P